Argo/Types/Decoded/Decoded.swift

```diff
@@ -82,8 +82,8 @@ public extension Decoded {
     - returns: A `Decoded.Failure` with a `.TypeMismatch` error constructed
                from the provided `expected` and `actual` values
   */
-  static func typeMismatch<T, U>(expected: String, actual: U) -> Decoded<T> {
-    return .Failure(.TypeMismatch(expected: expected, actual: String(actual)))
+  static func typeMismatch<T, U>(_ expected: String, actual: U) -> Decoded<T> {
```

Comment:
The other two seemed fine with the ignored label, this is the only one that I think I preferred the other way.